src/ldc/eh/msvc.d

```diff
@@ -377,14 +383,14 @@ void msvc_eh_terminate() nothrow
             mov RBX, 0xFFFFFF;
             and RDX, RBX;
             cmp RDX, 0xC48348;          // add ESP,nn  (debug UCRT libs)
-            je L_addESP_found;
+            je L_addESP_found;

```

Comment:
The mixture of line endings is probably my fault. Even with the editorconfig plugin VS seems to produce these quite often. I suppose the chrome editor has converted everything to unix-style (which is ok).